docs/contribution/30-overview/3-concepts/5-frontend-fragment.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Frontend Fragment
+---
+
+import OperationLink from "@site/src/components/OperationLink";
+
+The configuration of frontend fragments serves to integrate an extension’s frontend directly into defined [anchors](#anchors) within the mStudio.
+This is particularly useful when the extension’s frontend should be seamlessly integrated into mStudio to provide the feel of a native feature.
+
+Frontend fragments are configured on an extension.
+More detailed information can be found in the [Extension reference](../../../reference/extensions).
+
+Frontend fragments are embedded into mStudio via an iframe.
+This iframe is not rendered directly in the visible area of the page.
+Instead, the mStudio waits for communication via window messages that indicate which known components should be rendered.
+This approach ensures a consistent user experience and enables features such as overlays.
+
+Custom components and CSS cannot be used.
+To develop a frontend fragment, [Flow Remote Components](#flow-remote-components) must be used.
+
+Since frontend fragments are embedded via an iframe and Safari in particular is restrictive with third-party cookies, frontend fragments cannot rely on cookies.
+Authentication and session handling are done using [Session-Tokens](#authentifizierung-und-session-handling-mittels-session-token).
+
+## Anchors
+
+The mStudio provides clearly defined locations where frontend fragments can be integrated.
+These locations, called anchors, have a unique identifier and are configured in the extension.
```

Comment:
1. Wo bekomme ich die IDs her? Ja „Additionally, a feature is being developed to locate anchors directly“ aber jetzt?
2. Wo konfigurieren ich das? In meiner `yaml` vermutlich. Unter welchem Key? wie genau?

Reply:
Es gibt eine neue Anchor Reference Page, die darauf eingeht.
Wie ich das konfiguriere gehe ich im nächsten MR an, weil das Verwalten von Extensions nochmal komplett neu dokumentiert werden muss.